config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -450,6 +450,73 @@ presubmits:
         options:
         - name: ndots
           value: "1"
+
+  # An E2E test job to allow us to manually trigger the Venafi TPP E2E tests
+  # with the following GitHub comment:
+  #
+  #  /test pull-cert-manager-e2e-v1-20-venafi-issuer-tpp
+  #
+  # See https://github.com/jetstack/cert-manager/issues/3555
+  #
+  - name: pull-cert-manager-e2e-v1-20-venafi-issuer-tpp
+    cluster: gke
+    context: pull-cert-manager-e2e-v1-20-venafi-issuer-tpp
+    always_run: false
+    optional: true
+    max_concurrency: 4
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.3
```

Comment:
If this is a manual job, would we be better to match any/all branches here? Not a big issue though :)

Reply:
Ah, I missed that. I'll create a quick followup PR.